bin/prettier.js

```diff
@@ -232,6 +232,21 @@ if (stdin) {
 
     let output;
 
+    if (argv["list-different"]) {
+      try {
+        if (!prettier.check(input, options)) {
+          console.log(filename);
+          process.exitCode = 1;
+        }
+      } catch (e) {
+        // Add newline to split errors from filename line.
+        process.stdout.write("\n");
+        handleError(filename, e);
```

Comment:
we actually don't want to print the error here, we just want to print all the filenames that are different. It's up to the user to actually run prettier on all those files to see what's going on.